layouts/partials/head.html

```diff
@@ -22,6 +22,18 @@
 <meta property="og:description" content="{{ .Params.description }}">
 {{- end }}
 
+<!--favicons-->
+<link rel="apple-touch-icon" sizes="180x180" href="https://www.tidyverse.org/images/favicons/apple-touch-icon.png">
```

Comment:
Can't this just be `/images/favicons/apple-touch-icon.png`?

Reply:
Yup. I was just being super literal with the realfavicongenerator instructions… (>ლ)